dual-tokenomics/dual-dao/README.md

```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
 
 # Dual DAO
 
-Deposit DUAL to vote on Dual DAO proposals on [Realms](https://app.realms.today/dao/dual%20dao)\
+Dual DAO is hosted on [Realms](https://app.realms.today/dao/dual%20dao).\
 \
-Treasury earns fees on DIPs and exercised SOs.\
+The DAO enables participants to deposit DUAL tokens in return for elibility to vote on governace proposals. There is no minimum deposit size and there is no lock-up period for the deposit; users can withdraw their tokens at any time.\
```

Comment:
Add "In order to vote on proposals, DUAL must be deposited onto realms until proposals are finalized. Voting is currently set to 3 days. Quorum to pass any governance proposal is set to 1% of total token supply & 100,000 tokens are required to create a proposal."

Reply:
Re-worded with additions as suggested.